docs/PARAMETERS.md

```diff
@@ -62,8 +62,16 @@ Following options can be added to macro files, that are read by Geat4. Example f
  `/jpetmc/event/save2g`  
 * save event when 2g were registered (default false):  
  `/jpetmc/event/save3g`  
-  save event when 3g were registered (default false):  
+* save event when 3g were registered (default false):
   Options save2g/save3g  and saveEvtsDetAcc are separable !
+* save event with fixed minimal number of hits to save (default 0 means all):
+ `/jpetmc/event/saveEventWithMinSize [int]`
+* save event with fixed minimal number of hits to save (default 0 means all):
+ `/jpetmc/event/saveEventWithMaxSize [int]`
+* save event with fixed number of hits to save and within the given energy window (default -1 keV energy means all energies):
+The third argument is an additional option: 0 -> save all events in which there was N hits within energy window
+1 -> save only events that have only minimal fixed number of hits, which is in the energy window
+ `/jpetmc/event/setEnergyRange [min energy in keV - double] [max energy in keV - double] [bool]`
```

Comment:
I think the last parameter may be quite confusing to the users. maye it would be simpler to obtain the same functionality just by setting MinSize and MaxSize to the same value?